### DISCUSSION TITLE
How to create many to many relationship in prisma2.

### DISCUSSION THREAD

**Discussion by aakashkag** [prisma/prisma#2538] [Q&A] [answered]
I am using Prisma2+GraphQL(Nexus)

I have 4 tables
1)Oranization
2)User
3)Role
4)Assigned_Role

One user can have multiple roles in organization.
How we can assign new roles to user. In other word how we can create new record in  Assigned_Role table?
I am strgling to find solution for this. 
Thank you :)

model Oranization {
  oranization_id      Int      @default(autoincrement()) @id
  oranization_name    String
  Assigned_Role       Assigned_Role[]
}

model User {
  user_id          Int      @default(autoincrement()) @id
  user_first_name  String
  Assigned_Role    Assigned_Role[]
}

model Role {
  role_id       Int      @default(autoincrement()) @id
  role_name     String
  Assigned_Role Assigned_Role[]
}

model Assigned_Role {
  assigned_role_id    Int      @default(autoincrement()) @id
  role_id             Int      @default(autoincrement())
  user_id             Int      @default(autoincrement())
  organization_id     Int      @default(autoincrement())
  Organization        Int      @relation(fields: [organization_id], references: [organization_id])
  Role                Role     @relation(fields: [role_id], references: [role_id])
  User                User     @relation(fields: [user_id], references: [user_id])
}

Re: https://github.com/prisma/prisma/issues/1915
I saw this solution but didn't understood and also it doesn't solve my problem fully.

**Comment by aakashkag** [comment]
Correction:
 Organization Int @relation(fields: [organization_id], references: [organization_id])
To Organization Organization @relation(fields: [organization_id], references: [organization_id])

**Comment by aakashkag** [comment, marked as answer]
Finally I am able to created many to many relationship.
Here is repo I created to show many to many relationship.
https://github.com/aakashkag/prisma2_examples

**Comment by ludralph** [comment]
Hi @aakashkag,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
